index.js

```diff
@@ -0,0 +1,97 @@
+import { catsData } from "/data.js"
+
+const catsArray = []
```

Comment:
```suggestion
const cats = []
```

I wouldn't even put the type in the name when naming variables

Reply:
you aren't using this variable at all btw, you're redefining it in a scope further down. I'd just remove this

Reply:
Addressed, deleted